packages/mcdoc/src/runtime/checker/index.ts

```diff
@@ -832,21 +828,20 @@ function checkShallowly<T>(
 	const childDefinitions: (McdocType | undefined)[] = Array(children.length)
 		.fill(undefined)
 	const errors: McdocCheckerError<T>[] = []
-	if (typeDef.attributes) {
-		for (const attribute of typeDef.attributes) {
-			const handler = attributeHandlers[attribute.name]
-			if (handler) {
-				errors.push(
-					...handler(
-						runtimeNode.originalNode,
-						attribute,
-						simplifiedInferred,
-						typeDef,
-						options,
-					),
-				)
-			}
+	let assignable = true
+	handleAttributes(typeDef.attributes, options.context, (handler, config) => {
+		if (
+			handler.checkInferred?.(
+				config,
+				simplifiedInferred,
+				options.context,
+			) === false
+		) {
+			assignable = false
 		}
+	})
+	if (!assignable) {
+		errors.push({ kind: 'internal', node: runtimeNode })
```

Comment:
I'm not sure why you are introducing `internal` here. In general, it would be better to report a more specific error, maybe have the attribute not return a boolean here, but a McdocRuntimeCheckerError instead.

Reply:
I don't want the mcdoc checker to show an error here, because when the string attacher runs it will report its own (better) `undeclaredSymbol` warning. But I do still need the error condenser to discard possible types that have these errors indicated by attributes (mostly to support the `prefix=!` use case)